multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/validators/parameters/RouteValidatorTest.java

```diff
@@ -69,12 +69,11 @@ void testAttemptToCorrect(String inputRoute, Object noHostname, boolean isValid,
     }
 
     private Map<String, Object> constructContext(Object noHostname) {
-        Map<String, Object> context = new HashMap<>();
-        if (noHostname != null) {
-            context.put(SupportedParameters.NO_HOSTNAME, noHostname);
+        if (noHostname == null) {
+            return new HashMap<>();
```

Comment:
If you don't add entries in the map after that, you can return `Collections.emptyMap()`